config_scheduler.go

```diff
@@ -10,10 +10,16 @@ import (
 
 // SchedulerConfig holds relevant settings for the scheduler process.
 type SchedulerConfig struct {
-	TaskFinder           string  `bson:"task_finder" json:"task_finder" yaml:"task_finder"`
-	HostAllocator        string  `bson:"host_allocator" json:"host_allocator" yaml:"host_allocator"`
-	FreeHostFraction     float64 `bson:"free_host_fraction" json:"free_host_fraction" yaml:"free_host_fraction"`
-	CacheDurationSeconds int     `bson:"cache_duration_seconds" json:"cache_duration_seconds" yaml:"cache_duration_seconds"`
+	TaskFinder                    string  `bson:"task_finder" json:"task_finder" yaml:"task_finder"`
+	HostAllocator                 string  `bson:"host_allocator" json:"host_allocator" yaml:"host_allocator"`
+	FreeHostFraction              float64 `bson:"free_host_fraction" json:"free_host_fraction" yaml:"free_host_fraction"`
+	CacheDurationSeconds          int     `bson:"cache_duration_seconds" json:"cache_duration_seconds" yaml:"cache_duration_seconds"`
+	PlannerVersion                string  `bson:"planner_version" json:"planner_version" mapstructure:"planner_version"`
```

Comment:
for consistency we might want "PlannerVersion" to be just Planner, as it's the logical equivalent of TaskFinder and HostAllocator (and maybe we should reorder to make this more clear.

Reply:
Agreed.